enterprise/models.py

```diff
@@ -915,10 +915,27 @@ def get_paginated_content(self, query_parameters):
         Returns:
             dict: The paginated discovery service search results.
         """
+        results = []
         query = self.content_filter.copy()
         query.update(query_parameters)
         catalog_client = CourseCatalogApiServiceClient(self.enterprise_customer.site)
-        return catalog_client.get_search_results(query, traverse_pagination=False)
+        search_results = catalog_client.get_search_results(query, traverse_pagination=False)
+        for content in search_results['results']:
+            if content['content_type'] == 'courserun' and content['has_enrollable_seats']:
+                results.append(content)
+            elif content['content_type'] == 'program' and content['is_program_eligible_for_one_click_purchase']:
+                results.append(content)
+            elif content['content_type'] == 'course':
```

Comment:
I have update the PR. @douglashall I think, As we are filtering the `programs` and `courserun` but we should keep the `course` contents in pagination detail results at that point that may be useful to get some course information? thoughts?

Reply:
@asadiqbal08 As of now we are only supporting programs and course runs as content items in enterprise catalogs. This may change in the future, but for now we should remove this.